apis/core/lib/domain/csv/file-head.ts

```diff
@@ -8,7 +8,7 @@ export function loadFileHeadString(
   let partialLine = ''
   stream.on('readable', () => {
     let chunk
-    while ((chunk = stream.read()) !== null) {
+    while ((chunk = stream.read(1000)) !== null) {
```

Comment:
Force the reader to only read smaller chunks of the file at once.
Yes this number is a little bit random.